cockatrice/src/carddatabasemodel.cpp

```diff
@@ -152,8 +152,9 @@ bool CardDatabaseDisplayModel::filterAcceptsRow(int sourceRow, const QModelIndex
         return false;
 
     if (!cardName.isEmpty())
-        if (!info->getName().contains(cardName, Qt::CaseInsensitive))
-            return false;
+        if (!info->getName().contains(cardName, Qt::CaseInsensitive)) 
+            if (!CardInfo::simplifyName(info->getName()).contains(cardName, Qt::CaseInsensitive))
+                return false;
```

Comment:
https://github.com/poixen/Cockatrice/blob/card_search_punctuation/cockatrice/src/carddatabasemodel.cpp#L154-L157

Can we potentially refine this to
```
if (!cardName.isEmpty() && !info->getName().contains(cardName, Qt::CaseInsensitive) && !CardInfo::simplifyName(info->getName()).contains(cardName, Qt::CaseInsensitive))
     return false;
```